backend/src/components/fundingAgreements.js

```diff
@@ -116,11 +117,33 @@ async function getFundingReallocationRequests(req, res) {
   }
 }
 
+async function fundingAgreementExists(req, res) {
+  try {
+    const facilityIds = req.body?.facilityIds
+    const filter = facilityIds.map((id) => `(_ofm_facility_value eq ${id})`).join(' or ')
+    const operation = `ofm_fundings?$select=ofm_fundingid,ofm_funding_number,ofm_declaration,ofm_start_date,ofm_end_date,_ofm_application_value,_ofm_facility_value,statuscode,statecode,ofm_version_number&$filter=${encodeURIComponent(filter)}&$top=500`
```

Comment:
We don't need to select anything besides ofm_fundingid since we're not actually returning fas.

Reply:
This was addressed.